upgrade_dragonchain_uvn.sh

```diff
@@ -235,6 +233,11 @@ bootstrap_environment(){
     sudo snap refresh microk8s --channel=1.15/stable --classic >> $LOG_FILE 2>&1
     errchk $? "sudo snap refresh microk8s --channel=1.15/stable --classic >> $LOG_FILE 2>&1"
 
+    # Remove the snap package for helm and install direct with the known working version (just in case)
+    sudo snap remove helm --purge >> $LOG_FILE 2>&1
+
+    sudo curl -LO https://git.io/get_helm.sh >> $LOG_FILE 2>&1 && sudo bash get_helm.sh --version v2.14.3 >> $LOG_FILE 2>&1 && sudo rm get_helm.sh >> $LOG_FILE 2>&1
+    sudo helm init --history-max 200 --upgrade --force-upgrade >> $LOG_FILE 2>&1
```

Comment:
A few suggestions for consideration:

1.  Should we add the errchk feature to the snap/curl/helm init ? Is it helpful to know and log that output? Thinking especially if that fails, would identifying the problem be easier?
2.  I'm not a fan of the cmd1 && cmd2 but that is personal preference. I acknowledge it has its use cases. In this scenario, would it be best to split this into multiple lines and errchk each? What if you are unable to curl get_helm.sh? Do you want to log/abort ?
3. Shame on both of us - we should add both "wget" and "curl" to line 10 for required commands. wget is probably on most systems which is why it hasn't failed in any of these node installs. Not sure about curl as I'm not a heavy user.

Reply:
Done. :)